app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, new_file = command.split()
```

Comment:
First you should split to check that length is exactly 3

```
split_result = command.split()
if len(command.split()) == 3:
    cp, file, new_file = split_result
```


then make other checks